addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/DirectoryBrowsingScanRule.java

```diff
@@ -91,8 +96,8 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         String responsebody = msg.getResponseBody().toString();
 
         // try each of the patterns in turn against the response.
-        String evidence = null;
-        String server = null;
+        evidence = null;
+        server = null;
```

Comment:
I'm guessing this is a change your IDE made. While it isn't syntactically incorrect it also isn't relevant to this contribution.

Tiddying up things like this should be done more generally/specifically in it's own PR.

Reply:
This change is not correct either way, each scan uses its own scan rules.